src/shogun/mathematics/NormalDistribution.cpp

```diff
@@ -0,0 +1,97 @@
+#include <shogun/io/SGIO.h>
+#include <shogun/mathematics/NormalDistribution.h>
+
+#include <cmath>
+#include <limits>
+
+using namespace shogun;
+
+NormalDistribution::NormalDistribution(float64_t mean, float64_t stddev)
```

Comment:
the ctor could be constexpr, or?

Reply:
it should be possible

Reply:
nope, not possible. constexpr constructors must be default/deleted, or only contain typedefs